src/main/scala/com/linagora/openpaas/gatling/Configuration.scala

```diff
@@ -11,7 +11,8 @@ object Configuration {
   val OpenPaaSHostName = Properties.envOrElse("OPENPAAS_HOSTNAME", "localhost")
   val OpenPaaSPort = Properties.envOrElse("OPENPAAS_PORT", "8080").toInt
   val OpenPaaSProtocol = Properties.envOrElse("OPENPAAS_PROTOCOL", "http")
-  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
+//  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
+  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName"
```

Comment:
```suggestion
//  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName:$OpenPaaSPort"
  val OpenPaaSBaseUrl = s"$OpenPaaSProtocol://$OpenPaaSHostName"
```

I think we can ditch `OpenPaaSHostName`, `OpenPaaSPort`, `OpenPaaSProtocol` altogether. Something like this is enough, isn't it?

```
val OpenPaaSBaseUrl = Properties.envOrElse("OPENPAAS_BASE_URL", "http://localhost:8080")
```